compiler/src/iree/compiler/InputConversion/StableHLO/Passes.cpp

```diff
@@ -26,36 +26,19 @@ namespace {
 }  // namespace
 
 namespace {
-// TODO(#8745): remove these flags when the -iree-flow-demote-* flags can be
-// used without tripping upstream verifier issues.
-llvm::cl::opt<bool> clDemoteI64ToI32(
-    "iree-stablehlo-demote-i64-to-i32",
-    llvm::cl::desc(
-        "Converts all StableHLO i64 ops and values into i32 counterparts."),
-    llvm::cl::init(true));
```

Comment:
Removing this flag broke one of the samples: https://github.com/openxla/iree/actions/runs/5262963399/jobs/9512655180#step:3:258

Please search the full repo for uses of flags when changing them.

```python
 3 vm_flatbuffer = iree.compiler.tf.compile_module(
      4 TrainableDNN(),
      5 target_backends=[backend_choice],
      6 exported_names=exported_names,
      7 extra_args=["--iree-stablehlo-demote-i64-to-i32=false",
      8 "--iree-flow-demote-i64-to-i32"])

CompilerToolError: Error invoking IREE compiler tool iree-compile
Diagnostics:
iree-compile: Unknown command line argument '--iree-stablehlo-demote-i64-to-i32=false'.  Try: '/work/.notebook.venv/lib/python3.8/site-packages/iree/compiler/tools/../_mlir_libs/iree-compile --help'
iree-compile: Did you mean '--iree-flow-demote-i64-to-i32=false'?
```

Source: https://github.com/openxla/iree/blob/a219cb5008a40d16d9d6f67d1fcbca5e2bacba83/samples/colab/mnist_training.ipynb#L378-L390

Reply:
Sent a fix: https://github.com/openxla/iree/pull/14118